tests/yspryn/test_expressions.py

```diff
@@ -0,0 +1,24 @@
+def test_tuple_check():
+    a1 = 12,
+    a2 = (12)
+    a3 = (12,)
+
+    # identity check
+    assert a1 != a2
+    assert a1 == a3
+
+def test_create_check():
+    dict_var1 = dict
+    dict_var2 = {}
+    dict_var3 = dict()
+
+    assert dict_var1 != dict_var2
+    assert dict_var2 == dict_var3
```

Comment:
check that dict_var2 isinstance of dict_var1

check that if you call dict_var1 you get empty dict